File: 3.Source/Windows/MyDict.Admin/Language.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyDict.Admin.Entity
{
    public class Language
    {
        public uint ID { get; set; } 
        public string Name { get; set; }
        public string Title { get; set; }

        public Language()
        {
            ID = 0;
            Name = string.Empty;
            Title = string.Empty;
        }
    }
}

File: 3.Source/Web/MyDict.Services/language/language.php
<?php
namespace MyDict\Services\Entity;

defined("__MAGIC__") or die();

class Language {
    public $id = 0;
    public $name = '';
    public $title = '';

    public function __construct($id, $name, $title) {
        $this->id = $id;
        $this->name = $name;
        $this->title = $title;
    }
}
?>